style.css

```diff
@@ -136,3 +137,244 @@ h2 {
   order: 2;
   flex-grow: 0;
 }
+
+.cardworks-one {
+  box-sizing: border-box;
+
+  /* Auto layout  */
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+
+  /* Neutrals /N0 */
+  background: #ffff;
+
+  /* Neutrals /N40 */
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.cardworks-two {
+  box-sizing: border-box;
+
+  /* Auto layout  */
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 596px;
+
+  /* Neutrals /N0 */
+  background: #ffff;
+
+  /* Neutrals /N40 */
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.cardworks-three {
+  box-sizing: border-box;
+
+  /* Auto layout  */
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+
+  /* Neutrals /N0 */
+  background: #ffff;
+
+  /* Neutrals /N40 */
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.cardworks-four {
+  box-sizing: border-box;
+
+  /* Auto layout  */
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 596px;
+
+  /* Neutrals /N0 */
+  background: #ffff;
+
+  /* Neutrals /N40 */
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.frame {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 236px;
+  height: 32px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.tag {
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 214px;
+  height: 24px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.buttons {
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  position: absolute;
+  width: 126px;
+  height: 48px;
+  background-color: #fff;
+}
+
+.press {
+  width: 102px;
+  height: 24px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 5px;
+  border: 1px solid #396df2;
+  margin-top: 20px;
+  margin-bottom: 30px;
+  margin-left: 20px;
+
+  /* Identical to box-height or 141% */
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #396df2;
+
+  /* Inside auto layout */
+  flex: 0;
+  order: 0;
+  flex-grow: 0;
+}
```

Comment:
- [ ] Good job implementing some changes to button 🥇 However, as suggested by the reviewer [here](https://github.com/brendalutiku/Portfolio/pull/2#discussion_r1154248069) please make sure to stick to the figma design guidelines 🛠 See the images below for more clarity 👇

You design before 🔽 | Your design now 🔽 | Figma design 🔽
---| ---| ---|
![image](https://user-images.githubusercontent.com/106119105/229157422-a380a60b-6b19-4783-bd30-ab7db6a198fb.png) | ![image](https://user-images.githubusercontent.com/106119105/229157318-6e814152-9c3a-4c20-bd82-39edec455c78.png) | ![image](https://user-images.githubusercontent.com/106119105/229157587-31b7742e-8dc9-408c-8618-1209ee0e9df9.png)